akka-cluster/src/multi-jvm/scala/akka/cluster/RemoteFeaturesWithClusterSpec.scala

```diff
@@ -30,6 +30,7 @@ class ClusterRemoteFeaturesConfig(artery: Boolean) extends MultiNodeConfig {
     ConfigFactory.parseString(s"""
       akka.remote.log-remote-lifecycle-events = off
       akka.remote.artery.enabled = $artery
+      akka.remote.classic.netty.tcp.port = ${MultiNodeSpec.tcpPort}
       akka.remote.artery.canonical.port = ${MultiNodeSpec.tcpPort}
```

Comment:
This is a trick one. 

Previously, `akka.remote.classic.netty.tcp.port` was being set in `MultiNodeSpec` to the `selfPort`, but when running with aeron-udp, the selfPort will be 6000 (UDP). This is not what we want here. `akka.remote.classic.netty.tcp.port` should always be the tcpPort.

Btw, this needs to be enforced on MultiNodeSpec instead, just realising it now.